base/intfuncs.jl

```diff
@@ -1,3 +1,4 @@
+using Base: MultiplicativeInverses
```

Comment:
Should go after the license line

Reply:
Ha, my texteditor likes to insert lines like this.

Reply:
I don't trust the language server enough to be confident that line is actually necessary.